tests/test_username_api.py

```diff
@@ -40,306 +41,62 @@ def get_expected_response(website, user, status):
               else None
   }
 
+def load_test_cases(type):
+  res = []
+  for website in data:
+    if type == 'available_username':
+      usernames = data[website]['available_usernames']
+    if type == 'taken_username':
+      usernames = data[website]['taken_usernames']
+    if type == 'format_checking':
+      if website == 'facebook':
```

Comment:
you can do an append to ``usernames`` list ``if website == 'facebook'`` and bring the facebook condition down.

Reply:
hmm... do you mean to filter `facebook` at below?